client/app/vps/backup-storage/order/vps-order-backup-storage.html

```diff
@@ -2,42 +2,48 @@
 
     <div class="row d-lg-flex">
         <div class="col-xm-12">
-            <cui-wizard-form data-form-title="{{ 'vps_configuration_activate_ftpbackup_title' | translate }}"
-                             data-form-loaded="true"
-                             data-form-on-cancel="$ctrl.cancel()"
-                             data-form-on-complete="$ctrl.orderOption()">
+            <h2 class="oui-header_2">
+                {{ ::'vps_configuration_activate_ftpbackup_title' | translate }}
+            </h2>
 
-                <cui-wizard-step data-step-title="{{ 'vps_configuration_activate_ftpbackup_step1_question' | translate:{ serviceName: $ctrl.serviceName } }}"
-                                 data-step-loaded="$ctrl.model.optionDetails"
-                                 data-step-completed="true">
-                    <div class="text-center" data-ng-if="!$ctrl.model.optionDetails">
-                        <oui-spinner></oui-spinner>
+            <h3 class="oui-header_3" data-ng-bind-html="::'vps_configuration_activate_ftpbackup_step1_question' | translate:{ serviceName: $ctrl.serviceName }"></h3>
```

Comment:
`data-translate` and `data-translate-values` instead of `data-ng-bind-html`

Reply:
This is fixed :)